molgenis-jobs/src/main/java/org/molgenis/data/jobs/Job.java

```diff
@@ -31,18 +31,22 @@ public Job(Progress progress, TransactionTemplate transactionTemplate, Authentic
 	@Override
 	public Result call()
 	{
+		if (transactionTemplate != null)
+		{
+			return transactionTemplate.execute((status) -> doCallInTransaction());
+		}
+		else
+		{
+			return doCallInTransaction();
```

Comment:
Or do you mean the case where no transaction template is specified? There we don't run in a transaction.